### ISSUE TITLE
Save Functionality - Favorited poems are not displaying after favoriting

### ISSUE THREAD

**Issue by mattfromseattle** [suwebdev/great-sunrise#4]
**Issue:** Users must be able to unsave poems to their profile as "Favorites" to be displayed on demand. Currently, after a user marks a poem as a Favorite, it does not appear on their Favorite Poems list.

**User Story:**https://github.com/suwebdev/great-sunrise/wiki/Site-Visitor-User-Stories

**Priority:** HIGH
Due to this being a key piece of functionality, it is listed as HIGH priority.

**Considerations:**
- Database work should be in place prior to this work
    - Requires a new table for users saved poems
- User accounts/profile functionality should be in place
- Users can save poems in the following areas:
    - On the poem view
        - Users click on Favorites icon to save a poem
        - When marked, icon is filled and appears on users Favorite Poems list
    - On the Favorites view
        - Users can select multiple poems to remove from Favorites here
- Favorites view will need to be created to display favorite poems (https://material.angularjs.org/latest/demo/list) - **Should be a separate story**

**Orignal User Story:** As a visitor to the site, I want to be able to save poems so that I can easily access the ones that I like.

**Acceptance Criteria:**
- Poems have a favorite icon on them
- When a poem is favorited, icon will be filled
- If a poem is not favorited, icon will be empty
- Favorited poems will be listed in a Favorites section

**Estimated Size:** **Sm-Med** (1-5 days)

**Comment by mattfromseattle** [comment]
Pull Request #23 and #21 created to address this issue.
